route/build/src/public_var.f90

```diff
@@ -14,6 +14,8 @@ MODULE public_var
 
   ! physical constants
   real(dp),    parameter,public   :: pi=3.14159265359_dp    ! pi
+  real(dp),    parameter,public   :: Cw=4190.0_dp           ! heat capacity of water [J/kg/K]
```

Comment:
Check that these agree with the current values set in the shr_const_mod.F90 code under the share directory.